Form/Type/DoctrineORMSerializationType.php

```diff
@@ -137,10 +137,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         }
     }
 
+    // BC for SF < 3.0
```

Comment:
`{@inheritdoc}` should be added with your comment after it.

Please do it for other classes.